### DISCUSSION TITLE
How can I lower the label tab with no lip and keep height selected?

### DISCUSSION THREAD

**Discussion by Ruharc** [kennetek/gridfinity-rebuilt-openscad#109] [Q&A]
I want to be able to get rid of the lip so that it is just straight sides and keep the existing height which I am able to do. The problem is that now the label tab is all the way at the top level, with the top of the bin.

Is there a way to get rid of the lip, keep the height of the bin, and lower the tab to its normal position as if there was a lip?

Thank you everyone!  Loving this implementation of Gridfinity!

**Comment by Ruudjhuu** [comment]
this is currently not supported by using the gui parameters. It seems logical if you remove the stack-ability of a bin to use the space for storage.
